hdInsight-apache-spark-1-4-1/azuredeploy.json

```diff
@@ -0,0 +1,226 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "location": {
+            "type": "string",
+            "allowedValues": [
+                "Central US",
+                "East Asia",
+                "East US",
+                "Japan East",
+                "Japan West",
+                "North Europe",
+                "South Central US",
+                "Southeast Asia",
+                "West Europe",
+                "West US",
+                "East US 2"
+            ],
+            "metadata": {
+                "description": "The location where all azure resources will be deployed."
+            }
+        },
+        "clusterName": {
+            "type": "string",
+            "metadata": {
+                "description": "The name of the HDInsight cluster to create."
+            }
+        },
+        "LoginUsername": {
+            "type": "string",
+            "defaultValue": "hdiuser",
```

Comment:
Probably a good idea to remove a default username as a security best practice so that it isn't available on the open internet